src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Fruit {
+    private Map<String,Integer> fruitMap;

```

Comment:
it's better to create Storage in db package and replace map there

```suggestion
    private Map<String, Integer> fruitMap;
```